### ISSUE TITLE
Auto no longer works with npx

### ISSUE THREAD

**Issue by zephraph** [intuit/auto#440]
**Describe the bug**

Our releases started failing and it looks like `auto` doesn't work with npx past v7. 

```
#!/bin/bash -eo pipefail
npx auto shipit  $AUTO_ARGS
npm ERR! path /home/circleci/.npm/_npx/111/lib/node_modules/auto/node_modules/@auto-it/core/dist/bin/auto.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/circleci/.npm/_npx/111/lib/node_modules/auto/node_modules/@auto-it/core/dist/bin/auto.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/circleci/.npm/_logs/2019-05-19T08_17_24_669Z-debug.log
Install for auto@latest failed with code 254
Exited with code 254
```

**To Reproduce**

Run `npx --ignore-existing auto --help`

**Details**

Confirmed not to work with `npx 6.4.1`

**Comment by zephraph** [comment]
More info...

It actually fails on npm install. 

```
❯ npm i auto    
npm WARN test@1.0.0 No description
npm WARN test@1.0.0 No repository field.

npm ERR! path /Users/justinbennett/Git/oss/test/node_modules/@auto-it/npm/node_modules/@auto-it/core/dist/bin/auto.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/justinbennett/Git/oss/test/node_modules/@auto-it/npm/node_modules/@auto-it/core/dist/bin/auto.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/justinbennett/.npm/_logs/2019-05-19T16_30_36_527Z-debug.log
```

**Comment by adierkens** [comment]
:rocket: Issue was released in v7.0.2 :rocket:
